### ISSUE TITLE
Max pooling loss is slow

### ISSUE THREAD

**Issue by robotnc** [wenet-e2e/wekws#97]
I create a model of 100 keywords, and found that the max_pooling loss calculation is very slow. the loss calculation takes 8 second, loss.backward() takes about 16 second per batch(256).
Any plan to improve the speed of this loss ?

**Comment by robin1001** [comment]
It's a problem if there is so much keywords. We have no bandwith to do it recently.
